backend/src/providers/payments/razorpay.provider.ts

```
@@ -0,0 +1,354 @@
+    this.client = new Razorpay({ key_id: keyId, key_secret: keySecret });
+  }
+
+  /**
+   * Verify Razorpay webhook signature.
+   * Razorpay signs webhooks using HMAC-SHA256 of the raw body.
+   */
+  verifyWebhookSignature(rawBody: string, signature: string, webhookSecret: string): boolean {
+    const expected = crypto
+      .createHmac('sha256', webhookSecret)
+      .update(rawBody)
+      .digest('hex');
```

ESLint

[error] Replace `⏎······.createHmac('sha256',·webhookSecret)⏎······.update(rawBody)⏎······` with `.createHmac('sha256',·webhookSecret).update(rawBody)`

(prettier/prettier)